README.md

```diff
@@ -120,7 +120,7 @@ If you want to disable that behaviour set the following option in your config:
 
 Depending on the configured url you can initial the request through:
 
-    /auth/identity
+    /auth/identity/callback
```

Comment:
Why did you change this?

Reply:
@yordis  as per https://github.com/ueberauth/ueberauth_identity/issues/23#issuecomment-400173333 discussion.
Did I misunderstood it?

Reply:
and I keep forgetting that it is the `identity` strategy 🤦‍♂️